specification/policyinsights/resource-manager/readme.md

```diff
@@ -56,6 +62,14 @@ directive:
       - $.paths["/{scope}/providers/Microsoft.PolicyInsights/policyStates/$metadata"].get.produces[0]
 
 ```
+### Tag: package-2019-10
+
+These settings apply only when `--tag=package-2019-10` is specified on the command line.
+
+``` yaml $(tag) == 'package-2019-10'
+input-file:
+- Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json
```

Comment:
Should it include all related json files (like policyEvents etc. as that is defined in package-2018-07), to compose a complete package?

Reply:
Adding other json files to the package now.